cmake/compiler_flag_sets/native.cmake

```diff
@@ -0,0 +1,74 @@
+#
+# Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# C, CXX, LD, and AS flags for native targets.
+#
+
+#
+# Debug flags for C and C++
+#
+set(C_FLAG_SET
+                "-fprofile-arcs"
+                "-ftest-coverage"
+                "-Og"
+                "-DDEBUG"
+                "-ggdb"
+)
+
+#
+# Diagnostics for C and C++
+#
+list(APPEND C_FLAG_SET
+                "-pedantic"
+                "-Wall"
+                "-Wextra"
+                "-Werror"
+                "-Wfloat-equal"
+                "-Wconversion"
+                "-Wabi"
+                "-Wunused-parameter"
+                "-Wunused-variable"
+                "-Wunused-value"
+                "-Wcast-align"
+                "-Wmissing-declarations"
+                "-Wno-missing-field-initializers"
```

Comment:
Why?

Reply:
Same question here as above?

Reply:
changing to `-Wmissing-field-initializers`